src/include/index/ivf_pq_metadata.h

```diff
@@ -99,6 +100,7 @@ class ivf_pq_metadata : public base_index_metadata<ivf_pq_metadata> {
       {"sub_dimensions", &sub_dimensions_, TILEDB_UINT32, true},
       {"bits_per_subspace", &bits_per_subspace_, TILEDB_UINT32, true},
       {"num_clusters", &num_clusters_, TILEDB_UINT32, true},
+      {"distance_metric", &distance_metric_, TILEDB_UINT32, false},
```

Comment:
I think for IVF PQ we can make this required as we don't have any indexes released yet:
```suggestion
      {"distance_metric", &distance_metric_, TILEDB_UINT32, true},
```
Though Vamana we'll need to leave it optional.